addons/metrics-server/template/testgrid/k8s-docker.yaml

```diff
@@ -0,0 +1,11 @@
+kubernetes:
+  version: "latest"
+weave:
+  version: "latest"
+rook:
+  version: "latest"
+docker:
+  version: "latest"
+metricsServer:
+  version: "__testver__"
+  s3Override: "__testdist__"
```

Comment:
This will get run against new or updated versions of this addon. You should be able to add more than one template file per directory.